test/brsTypes/components/RoSGNode.test.js

```diff
@@ -235,22 +244,36 @@ describe("RoSGNode", () => {
                 expect(keys).toBeTruthy();
 
                 let result = keys.call(interpreter);
-                expect(result.elements).toEqual(new RoArray([cletter, letter1, letter2]).elements);
+                expect(result.getElements()).toEqual(
+                    new RoArray([change, cletter, focusable, focusedChild, id, letter1, letter2])
+                        .elements
+                );
             });
 
             it("returns an empty array from an empty associative array", () => {
+                let change = new BrsString("change");
+                let focusable = new BrsString("focusable");
+                let focusedChild = new BrsString("focusedchild");
+                let id = new BrsString("id");
+
                 let node = new RoSGNode([]);
 
                 let keys = node.getMethod("keys");
                 expect(keys).toBeTruthy();
 
                 let result = keys.call(interpreter);
-                expect(result.elements).toEqual(new RoArray([]).elements);
+                expect(result.getElements()).toEqual(
+                    new RoArray([change, focusable, focusedChild, id]).elements
```

Comment:
Does "new RoArray([...]).elements" wrap the elements in an array and then immediately unwrap this?  Why can't we just pass in the array itself to toEqual()?

Reply:
True